test/runtests.jl

```diff
@@ -1,3 +1,6 @@
+using MATLAB
+using Base.Test
```

Comment:
I would leave this in the individual test files. It's sometimes useful to be able to execute them without running the rest of the tests.